Dockerfile

```
@@ -0,0 +1,50 @@
+FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
+WORKDIR /app
+
+COPY . .
+RUN dotnet restore
+WORKDIR ./src/Presentation
+RUN dotnet publish -c Release -o build  --no-restore --verbosity m
+
+FROM mcr.microsoft.com/dotnet/aspnet:7.0
+WORKDIR /app
+
+RUN groupadd -r -f librum-server
+RUN useradd -r -g librum-server -d /var/lib/librum-server --shell /usr/sbin/nologin librum-server
+
+
+RUN mkdir -p /var/lib/librum-server/srv
+COPY --from=build /app/src/Presentation/build/* /var/lib/librum-server/srv
+COPY --from=build /app/appsettings.json /var/lib/librum-server/srv
```

I had to change this as follows:

```suggestion
RUN mkdir -p /var/lib/librum-server/
COPY --from=build /app/src/Presentation/build /var/lib/librum-server/srv
COPY --from=build /app/appsettings.json /var/lib/librum-server/srv/
```

Otherwise, on Docker version 24.0.5, I get:

```
Step 12/39 : COPY --from=build /app/src/Presentation/build/* /var/lib/librum-server/srv
When using COPY with more than one source file, the destination must be a directory and end with a /
```

---

That's fine, it's working for me on Docker 24.0.7, build afdd53b. If this is more robust and achieves the same, then it's better to do it this way.